### PR TITLE
DO-297: Dashboard styling fixed.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/common/DashboardSmallCard/DashboardSmallCard.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const {
+    heading,
+    body,
+    link,
+    iconClass
+  } = props;
+
+  return (
+    <Container className="dashboard-small-card">
+      <div className="dashboard-small-card__heading">
```
Comment (mohitsaladiya) [inline src/components/common/DashboardSmallCard/DashboardSmallCard.jsx:24]: Create component for DashboadSmallCardHeading

File: src/components/common/DashboardSmallCard/DashboardSmallCard.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        <span className="dashboard-small-card__heading-text">
+          {heading}
+        </span>
+      </div>
+      <div className="dashboard-small-card__body">
```
Comment (mohitsaladiya) [inline src/components/common/DashboardSmallCard/DashboardSmallCard.jsx:29]: Create component for DashboadSmallCardBody

File: src/components/common/DashboardSmallCard/DashboardSmallCard.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <div className="dashboard-small-card__body-content">
+          <span className={`icon ${iconClass}`}>{body}</span>
+        </div>
+        <div className="dashboard-small-card__navigation">
```
Comment (mohitsaladiya) [inline src/components/common/DashboardSmallCard/DashboardSmallCard.jsx:33]: Create component for DashboadSmallCardNavigationLink

File: src/components/member/MembersOccasion/OccasionsCountRow/OccasionsCountRow.jsx [hunk 4]
```diff
@@ -14,9 +14,9 @@ import { TIME_PERIOD } from '../../../../constants/common/commonConstants';
  */
 const OccasionsCountRow = props => (
   <Container className="occasions-count-row">
-    <OccasionCount count={props.todayCount} timePeriod={TIME_PERIOD.TODAY} />
-    <OccasionCount count={props.weekCount} timePeriod={TIME_PERIOD.WEEK} />
-    <OccasionCount count={props.monthCount} timePeriod={TIME_PERIOD.MONTH} />
+    <DashboardCalendarCard count={props.todayCount} name={TIME_PERIOD.TODAY} />
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/OccasionsCountRow/OccasionsCountRow.jsx:17]: timePeriod should be the correct prop name here.
```suggestion
    <DashboardCalendarCard count={props.todayCount} timePeriod={TIME_PERIOD.TODAY} />
```

File: src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * This component display the count of occasion along with the time period.
+ * Like '2 today', '5 This month' .
+ * @param {Object} props
+ * @return {*}
+ * @constructor
+ */
+const DashboardCalendarCard = props => (
+  <Container className="dashboard-calendar-card">
+    <div className="dashboard-calendar-card__heading">
```
Comment (mohitsaladiya) [inline src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx:16]: Create a child component for DashboardCalendarCardHeading

File: src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      <span className="dashboard-calendar-card__heading-text">{props.name}</span>
+    </div>
+    <div className="dashboard-calendar-card__body">
```
Comment (mohitsaladiya) [inline src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx:19]: Create a child component for DashboardCalendarCardBody

Comment (amitjain24524) [reply src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx]: done

File: src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx (hunk 5 above)
Comment (amitjain24524) [reply src/components/common/DashboardCalendarCard/DashboardCalendarCard.jsx]: done

File: src/components/member/MembersOccasion/OccasionsCountRow/OccasionsCountRow.jsx (hunk 4 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/OccasionsCountRow/OccasionsCountRow.jsx]: done

File: src/components/common/DashboardSmallCard/DashboardSmallCard.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/common/DashboardSmallCard/DashboardSmallCard.jsx]: done

File: src/components/common/DashboardSmallCard/DashboardSmallCard.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/DashboardSmallCard/DashboardSmallCard.jsx]: done

File: src/components/common/DashboardSmallCard/DashboardSmallCard.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/common/DashboardSmallCard/DashboardSmallCard.jsx]: done

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuNavigationLinks/AppHeaderMenuNavigationLinks.jsx [hunk 7]
```diff
@@ -12,7 +14,11 @@ import DashboardIconLink from '../../../../../member/DashboardIconLink/Dashboard
  */
 const AppHeaderMenuNavigationLinks = () => (
   <Container className="app-header-navigation-links">
-    <DashboardIconLink />
+    <span>
```
[the remark this answers is not in the document]
Comment (amitjain24524) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuNavigationLinks/AppHeaderMenuNavigationLinks.jsx]: done

**Pull request merged**